### ISSUE TITLE
payload examples in various tests may be wrong

### ISSUE THREAD

**Issue by simylein** [truvami/decoder#24]
our current test suites has the following inconsistencies

<h3>tagsl</h3>

<h4>port 1</h4>
there is a payload with 30 bytes, however the port only allows for 17 bytes somehow? <a href="https://docs.truvami.com/docs/payloads/tag%20S/v2.1.0">docs</a>
<code>0002c420ff005ed85a12b4180719142607240001ffbaffc2fc6f09a71d2e</code>

<h4>port 105</h4>
there is a payload with 56 bytes, however the port only allows for 49 bytes somehow? <a href="https://docs.truvami.com/docs/payloads/tag%20S/v3.2.0">docs</a>
<code>0028672658500172a741b1e238b572a741b1e08bb03498b5c583e2b172a741b1e0cda772a741beed4cc472a741beef53b772a741b1dd0000</code>

**Comment by michaelbeutler** [comment]
@michaelbeutler check port 1 example source.

**Comment by simylein** [comment]
> I'm unable to locate the source for the port 1 example. However, there was an issue with the 105 payloads that has been fixed at a firmware level and should now be fixed.

Yes, because I fixed that in the giant payload validation pull request, you would need to checkout the repository at some commit before that.
